content/api/commands/children.md

```diff
@@ -52,7 +52,8 @@ Pass in an options object to change the default behavior of `.children()`.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`.children()` yields the new DOM element(s) it found.</li></List>
+- `.children()` yields the new DOM element(s) it found.
+- `.children()` is a query, and it is _safe_ to chain further methods.
```

Comment:
```suggestion
- `.children()` is a query, and it is _safe_ to chain further commands.
```

Reply:
Should these all be consistent between the docs?

Reply:
The "unsafe" verbage has further qualification because it's unsafe in *some cases*, and I wanted to call out what those cases are (queries or commands that rely on the subject).

Here though, it's just always safe. You can chain any method (query, command or assertion) off of a query, whether or not it relies on the subject. I don't think "it's always safe" needs further qualification.

Reply:
I've rewritten these "safe/unsafe" sections, so going to resolve all these comments and we can start fresh threads if there are still more changes we want to see.